com.unity.multiplayer.mlapi/Runtime/Messaging/RpcParams.cs

```diff
@@ -2,6 +2,7 @@ namespace MLAPI.Messaging
 {
     public struct ServerRpcSendParams
     {
+        public NetworkUpdateManager.NetworkUpdateStages UpdateStage;
```

Comment:
I think we need an RFC for this and the other change below. I will be working an RFC that extends RPC API with things we want to bring over from existing MLAPI stuff, I might make these changes a part of it. Are these absolutely required in this PR? If not, leave it up to me and we can implement this functionality later.

Reply:
(or instead, we can revisit this separately later!)